src/js/apps/patients/patient/sidebar/sidebar_app.js

```diff
@@ -15,9 +19,27 @@ export default App.extend({
     return Radio.request('entities', 'fetch:patient:engagementStatus', patient.id);
   },
   onStart() {
-    this.showChildView('engagement', new EngagementStatusView({ model: this.patient }));
+    this.showEngagementStatus();
   },
   onFail() {
-    this.showChildView('engagement', new EngagementStatusView({ model: this.patient }));
+    this.showEngagementStatus();
+  },
+  showEngagementStatus() {
+    const engagementStatusView = this.showChildView('engagement', new EngagementStatusView({ model: this.patient }));
+
+    this.listenTo(engagementStatusView, 'click', this.showEngagementSidebar);
+  },
+  showEngagementSidebar() {
```

Comment:
maybe this should go into the `services/sidebar`?  Otherwise when you open another sidebar and close it, it will return the engagement sidebar instead of the patient one.. not sure that's ideal.

Reply:
Ah, I was wondering where the best place for this was. I missed that use case. Services seems like a good idea.